.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -14,11 +14,21 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
+        os: [windows-latest, ubuntu-latest]
         bwc_version : [ "2.0.1", "2.1.0", "2.2.1", "2.3.0", "2.4.1", "2.5.0-SNAPSHOT" ]
         opensearch_version : [ "3.0.0-SNAPSHOT" ]
+        exclude:
+          - os: windows-latest
+            bwc_version: "2.0.1"
+          - os: windows-latest
+            bwc_version: "2.1.0"
+          - os: windows-latest
+            bwc_version: "2.2.1"
+          - os: windows-latest
+            bwc_version: "2.3.0"
```

Comment:
is it possible to use list for exclude bwc_version here? It should minimize chance of error while maintaining with new release

Reply:
I have tried. It is failing the workflow
https://github.com/naveentatikonda/k-NN/actions/runs/3906264069/workflow

Reply:
have you tried syntax similar to bwc_versions?
```
- os: windows-latest
   bwc_version: [ "2.0.1", "2.1.0", "2.2.1", "2.3.0" ]
```

Reply:
In this case, it is not working as expected. It is not excluding those versions and triggering github checks for those versions("2.0.1", "2.1.0", "2.2.1", "2.3.0") as well for windows OS
https://github.com/naveentatikonda/k-NN/actions/runs/3906822394